spec/models/vat_rate_spec.rb

```diff
@@ -11,16 +11,12 @@
 
 require 'rails_helper'
 
-describe VatRate do
-
-  before(:all) do
-    @vr1 = FactoryBot.create :vat_rate, effective_date: 1.year.ago, rate_base_points: 2225
-    @vr2 = FactoryBot.create :vat_rate, effective_date: 3.years.ago, rate_base_points: 800
-    @vr3 = FactoryBot.create :vat_rate, effective_date: 10.years.ago, rate_base_points: 1750
-  end
-
-  after(:all) do
-    VatRate.destroy( [ @vr1.id, @vr2.id, @vr3.id ] )
+RSpec.describe VatRate do
+  before do
+    VatRate.delete_all
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2271/files#r173426782)